src/Scripting/GraphTypes.hpp

```
@@ -0,0 +1,217 @@
+			std::string label;
+			std::type_index type;
+		};
+
+		// Data Input Pin
+		struct DataInPin : Pin {
+			AsTL::StackValue val;					// Value (could be default value if there is no data link to this pin)
+		};
+
+		// Combo-box Input Pin
+		// NOTE: The combo type is a variable combo box filter (e.g., type is BOOL => only BOOL variables are shown in the dropdown)
+		struct ComboInPin : Pin {
```

Clang (14.0.6)

[warning] constructor does not initialize these fields: chosenVar

(cppcoreguidelines-pro-type-member-init)